components/SerialServer/include/plat/hifive/plat/serial.h

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Are you happy to instead treat the arm_common files as arm/riscv common and update the CMake scripts accordingly?

Reply:
yeah that sounds good